modules/c++/mem/include/mem/VectorOfPointers.h

```diff
@@ -116,10 +117,24 @@ class VectorOfPointers
     iterator end() { return mValues.end(); }
     const_iterator end() const { return mValues.end(); }
 
+    iterator erase(const iterator &pos)
```

Comment:
Iterators can be passed in by value rather than const ref (they're passed in by value throughout the STL)